exercise/do-your-own-curry.js

```diff
@@ -0,0 +1,49 @@
+/*
+## Do your own curry (function)
+
+### Instructions
+
+Create the function Curry that will curry any functions put as argument.
+
+example:
+
+const Mult2 = (el1,el2) => el1 * el2
+console.log(Mult2(2,2)) // result epected 4
+
+
+const Mult2Curried = Curry(Mult2)
+
+console.log(Mult2Curried(2)(2)) // result expected 4 (same result,
+with a function that has technically only one argument)
+
+### Notions
+
+-currying
+
+https://stackoverflow.com/questions/36314/what-is-currying
+
+*/
+
+///*/ // ⚡
```

Comment:
both delimiters are badly formated here and will cause problems

this is the proper format:
```js
// /*/ // ⚡
```
Spaces matters